Dockerfile

```diff
@@ -2,16 +2,25 @@ FROM --platform=linux/amd64 python:3.10-slim
 
 # Set pip to have no saved cache
 ENV PIP_NO_CACHE_DIR=false \
-    POETRY_VIRTUALENVS_CREATE=false
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
+    POETRY_VERSION=1.1.15 \
+    POETRY_HOME="/opt/poetry" \
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    POETRY_NO_INTERACTION=1 \
+    INSTALL_DIR="/opt/dependancies" \
```

Comment:
`dependancies` seems like a misspelling?

Reply:
Thanks for your keen eye. This has been fixed.